api/jobs.yaml

```diff
@@ -200,6 +200,9 @@ definitions:
         type: array
         items:
           type: string
+      primary:
```

Comment:
A `description` that this field controls whether or not the field should be displayed in the column view would be useful here!